x-pack/test/api_integration/apis/monitoring/apm/instance_mb.js

```diff
@@ -12,7 +12,10 @@ import { getLifecycleMethods } from '../data_stream';
 export default function ({ getService }) {
   const supertest = getService('supertest');
 
-  describe('instance detail mb', () => {
+  describe('instance detail mb', function () {
```

Comment:
Is there a reason for swapping to function, rather than arrow function?

Reply:
> Wouldn't it be nice if we could easily regenerate archives on the fly to account for different contexts? :D
I see what you did there ;)

> Is there a reason for swapping to function, rather than arrow function?
As far as I know you need `this` in order to add test tags. But I could switch it if you know another way to add the tags.